packages/material-ui/src/RadioGroup/RadioGroup.test.js

```diff
@@ -257,6 +260,61 @@ describe('<RadioGroup />', () => {
     });
   });
 
+  describe('useRadioGroup', () => {
```

Comment:
I copied the `useFormControl` tests and amended them accordingly: https://github.com/mui-org/material-ui/blob/55e12c9c9530c28ac98a94ed656feef6099569a2/packages/material-ui/src/FormControl/FormControl.test.js#L182-L291